src/can.rs

```diff
@@ -163,6 +163,67 @@ pub mod config {
         }
     }
 
+    /// Interrupt allocation flags.
+    /// These flags can be used to specify which interrupt qualities the code calling esp_intr_alloc* needs.
+    #[derive(Debug, Copy, Clone, Eq, PartialEq)]
+    pub enum IntrFlags {
```

Comment:
did you check if all of the flags are present on all platforms? e.g on esp32, esp32s2 ,esp32s3,  esp32c3

Reply:
Good point, I have double-checked the documentation and it looks like those flags are present on all platforms.